ui.apps/src/main/content/jcr_root/apps/themecleanflex/components/page/renderer.html

```diff
@@ -8,6 +8,12 @@
 
 <script src="/etc/felibs/themecleanflex/dependencies/intersection-observer.js"></script>
 <script src="/etc/felibs/themecleanflex/dependencies/v-lazy-image.min.js"></script>
+<script src="/etc/felibs/themecleanflex/dependencies/sha256.js"></script>
+<script src="/etc/felibs/themecleanflex/dependencies/vue-2.6.js"></script>
```

Comment:
this is an issue, we really should instead upgrade the pagerendervue version in the main peregrine project

Reply:
I'm fine with it but I still think it's a huge flaw in the architecture. That the theme relies on the main project

Reply:
@Felix-Puetz to clear this up a bit: themeclean-flex relies on peregrine-cms/pagerenderer/vue for the base functionality of how to render a page - a theme can be fully independent but then it needs to implement the functionality provided by peregrine-cms/pagerender/vue - the admin console also builds on peregrine-cms/pagerenderer/vue - themecleanflex declares its dependency on pagerendervue at https://github.com/headwirecom/themeclean-flex/blob/develop/ui.apps/src/main/content/jcr_root/content/themecleanflex/.content.xml